### PR TITLE
libsql: Add Connection::total_changes() API

### PR DESCRIPTION
**Pull request by penberg**
It's a rusqlite API that we're missing.

### CODE REVIEW

File: libsql/src/hrana/connection.rs [hunk 1]
```diff
@@ -54,6 +54,10 @@ where
         self.current_stream().affected_row_count()
     }
 
+    pub fn total_changes(&self) -> u64 {
+        self.current_stream().total_changes()
```
Comment (haaawk) [inline libsql/src/hrana/connection.rs:58]: What happens when the stream changes on connection?
Hrana allows multiple streams on a single connection - how does that fit with this?

Comment (haaawk) [reply libsql/src/hrana/connection.rs:58]: ok I see that `total changes` comes from server and is stream independent, correct?

Comment (haaawk) [reply libsql/src/hrana/connection.rs:58]: Ok it actually does not come from the server. Shouldn't we accummulate the counter in connection not in stream?

**Comment by haaawk** [comment]
LGTM in general but I think for HRANA we should store the counter in connection not in stream.

File: libsql/src/hrana/connection.rs (hunk 1 above)
Comment (penberg) [reply libsql/src/hrana/connection.rs:58]: You are right, but it's a bit involved to fix it. I opened https://github.com/tursodatabase/libsql/issues/1512 for tracking this, but merging this as intermediate solution.

**Pull request merged**